index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Website</title>
+</head>
+<body>
+    <header class="container">
+        <a class="logo">My Logo</a>
+
+        <nav class="menu-icon">
+            <img src="src/Icon-Menu.png" alt="menu"/>
+        </nav>
+    </header>
+
+    <section class="heading">
+
+        
+            <img class="ellipse3" src="src/Ellipse3.png" alt="ellipse"/>
+            <img class="rectangle4" src="src/Rectangle4.png" alt="rectangle4"/>
+            <img class="ellipse5" src="src/Ellipse3.png" alt="ellipse"/>
+            <img class="rectangle7" src="src/Rectangle4.png" alt="rectangle4"/>
+            <img class="ellipse11" src="src/Ellipse3.png" alt="ellipse"/>
+            <img class="ellipse12" src="src/Ellipse12.png" alt="ellipse12"/>
+            <img class="ellipse13" src="src/Ellipse13.png" alt="ellipse13"/>
```

Comment:
[OPTIONAL] Great work on adding a background image as requested by the previous code reviewer :+1: . However, you can improve this one by following a few steps below:

- [ ] As it is a background image,  adding it using `CSS` property `background-image` would make much since HTML `img` tag is used when the image has a value or is related to the content. In our case, it is just a background image and has no value to the content. Please refer to the below sample code:
```CSS
selector {
  background-image: url(image_path);
  background-repeat: no-repeat;
  background-size: contain;
  }
```

- [ ] Please make sure you choose the correct `selector`, in this case it should `.heading`
- [ ] Plese note that you can extract whole bg image from the figma. This time I did it for you :relaxed: please go a head and [download here](https://drive.google.com/file/d/1D_lhIyVm1KBknhxVo4-d_uYRYN0_iHTq/view?usp=sharing). Also please make sure you know how to extract the whole image on your own :pray: .

More on that you can find [here](https://www.w3schools.com/cssref/pr_background-image.asp)